k8s/gcp-pubsub/turbinia-output-claim-filestore.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: output-claim
+spec:
+  # Specify "" as the storageClassName so it matches the PersistentVolume's StorageClass.
+  # A nil storageClassName value uses the default StorageClass. For details, see
+  # https://kubernetes.io/docs/concepts/storage/persistent-volumes/#class-1
+  accessModes:
+  - ReadWriteMany
+  storageClassName: ""
+  volumeName: output
+  resources:
+    requests:
+      storage: 1T
```

Comment:
So what is this doing exactly?  Is this allocated the storage for GKE specifically?

Reply:
Yep in order to work with storage in Kubernetes you need a `PersistentVolume` or `StorageClass` (just another way of creating a `PersistentVolume` and a `PersistentVolumeClaim` which is essentially a token to use the `PersistentVolume`. The storage is a required attribute for both and they need to match, more details here: https://kubernetes.io/docs/concepts/storage/persistent-volumes/